src/Symfony/Bundle/FrameworkBundle/Command/AbstractConfigCommand.php

```diff
@@ -59,7 +60,7 @@ protected function listBundles($output)
     /**
      * @return ExtensionInterface
      */
-    protected function findExtension($name)
+    protected function findExtension($name, ContainerBuilder $container)
```

Comment:
that's a BC break

Reply:
My bad!

I don't remember, is it ok to add a new deprecation in a patch version?

Reply:
My bad!

I don't remember, is it ok to add a new deprecation in a patch version?

Reply:
We cannot do that in a patch release.

Reply:
Fixed in https://github.com/symfony/symfony/pull/46434